src/components/Preferences/OtherSettings.tsx

```diff
@@ -9,21 +9,26 @@ export default function OtherSettings() {
 
   const [pageSize, setPageSize] = useRecoilState(pageSizeState);
 
-  const pageSizeOptions = AVAILABLE_PAGE_SIZES.map(s => ({
-    key: s.toString(),
-    text: s.toString(),
-  }));
+  const onChange = (event: any) => {
+    const selectedSize = event.detail.selectedOption.value;
+    setPageSize(parseInt(selectedSize));
+  };
 
   return (
     <div className="preferences-row">
       <span className="fd-has-color-status-4">
         {t('settings.other.default-page-size')}
       </span>
-      <Select
-        options={pageSizeOptions}
-        selectedKey={pageSize.toString()}
-        onSelect={(_, { key }) => setPageSize(parseInt(key))}
-      />
+      <Select onChange={onChange}>
+        {AVAILABLE_PAGE_SIZES.map(available_size => (
+          <Option
+            value={available_size.toString()}
+            selected={pageSize.toString() === available_size.toString()}
+          >
+            {available_size.toString()}
```

Comment:
are all of these two `toString` necessary? I check that and everything is working without this.